src/data/profiles.js

```diff
@@ -144,6 +144,71 @@ const officers = [
 		programming, I like gaming, playing bass, watching anime and rewatching Gilmore Girls,
 		and spending way too much time on Discord. You can always find me losing sleep from
 		procrastination.`
+	},
+	{
+		role: 'Intern',
+		name: 'Max Akira Lee',
+		pronouns: 'he/him',
+		id: 'max',
+		description: `Hey! My name is Max and I'm a first year CS student. I love Hack! 
+		I also love working out, playing soccer, anything all you can eat, bumping Yeat, watching Kdramas/Jmovies, and Bplate.`
```

Comment:
What in the world is bumping yeat i even googled it 😭

Reply:
<img width="1140" alt="Screenshot 2024-02-26 at 11 39 34 AM" src="https://github.com/uclaacm/hack.uclaacm.com/assets/41168140/a7f3d94e-a125-4509-b0dd-c9c06ce558cb">

smh james. simple google search